lib/config.js

```diff
@@ -535,4 +555,14 @@ export const MINISTER_CLASSIFICATIONS = {
     // Protestants
     "http://lblod.data.gift/concepts/83d50e9184ae4a628851370079d162f6", // Predikant
   ],
+  "http://lblod.data.gift/concepts/cbd4c767ea184b41bf51b65d406754d4": [
+    //Niet-confessioneel
+    "http://lblod.data.gift/concepts/ea58c73d9b4fc8a24a4b3eaa73a33995", // Coördinator
```

Comment:
I'm not sure what i should've put here. these 2 options seemed reasonable to me since Niet-confessioneel is not really a religion with Priests or anything.